maintainer/conda/environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - chemfiles>=0.10
   - codecov
   - cython
+  - docutils <0.18
```

Comment:
If the issue is docutils, is the sphinx pin also necessary? (or is sphinx not backwards compatible here?)

Reply:
It's more of a cautionary pin -- we already know that sphinx_rtd_theme doesn't work with >=7. Up until now we've just happened to install sphinx<7 in both our CI (6.2.1 in PR 4198, 5.0.2 in RTD) but relying on that seemed fragile.